types/aws-lambda/trigger/sns.d.ts

```diff
@@ -24,6 +24,7 @@ export interface SNSMessage {
     UnsubscribeUrl: string;
     TopicArn: string;
     Subject: string;
+    Token?: string;
```

Comment:
If AWS will always send this, don't make it optional.

Reply:
Thanks @simonbuchan. Can confirm this property is optional.